src/coreclr/jit/compiler.cpp

```diff
@@ -4847,6 +4847,10 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
     //
     DoPhase(this, PHASE_MARK_LOCAL_VARS, &Compiler::lvaMarkLocalVars);
 
+    // Dominator and reachability sets are no longer valid.
+    fgDomsComputed         = false;
+    fgCompactRenumberQuirk = true;
+
```

Comment:
`lvaMarkLocalVars` uses `Compiler::IsDominatedByExceptionalEntry` to set `LclVarDsc::lvVolatileHint`. That is used by VN-based copy prop as part of the heuristic. My next step will be to compute SSA's dominators after unrolling above and then use those dominators to compute this heuristic instead. That will allow us to move `fgDomsComputed` up to where the loop table is invalidated as well.